packages/nextjs/app/page.tsx

```diff
@@ -1,7 +1,6 @@
 import "./page.css";
 import type { NextPage } from "next";
 import Leaderboard from "~~/components/onchain-impact-dashboard/Leaderboard";
-import { TEST_PROJECTS } from "~~/utils/onchainImpactDashboard/initialMockData";
```

Comment:
Can we just get rid of the entire `initialMockData.ts` file now?

Reply:
yes, we have now the endpoints connected and the mock data on the api side

Reply:
also now we have data for 337 projects in the mocked one